apps/dialtone-documentation/docs/.vuepress/theme/constants/themes.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Theme System Constants
+ *
+ * Single source of truth for all theme-related metadata in the Dialtone documentation site.
+ * This file defines the complete list of available themes, their display names, and
+ * organizational structure for UI presentation.
+ *
+ * @module theme/constants/themes
+ */
+
+/**
+ * DOM element IDs used for dynamic style tag injection
+ * @constant {Object} STYLE_TAG_IDS
+ */
+export const STYLE_TAG_IDS = {
+  BRAND_OVERRIDE: 'dialtone-css-brand-override',
+  CONTRAST: 'dialtone-css-contrast',
+};
+
+/**
+ * Complete list of theme IDs in display order.
+ * This order must match the import sequence in client.js for proper theme loading.
+ *
+ * Structure:
+ * 1. Base theme (dp) - Foundation for all other themes
+ * 2. Partner themes (tmo) - Brand partner customizations
+ * 3. Color assistive themes (prota-deuter, trita) - Accessibility variants
+ * 4. Named themes - Production-ready themes with marketing names (alphabetical)
+ * 5. Numbered themes (101-137) - Experimental themes awaiting final names
+ *
+ * @constant {string[]} ALL_THEME_IDS
+ */
+export const ALL_THEME_IDS = [
+  // Base theme
+  'dp',
+  // Partner themes
+  'tmo',
+  // Color assistive themes (accessibility)
+  'prota-deuter',
+  'trita',
+  // Named themes (alphabetical)
+  'aegean',
+  'botany',
+  'buttercream',
+  // 'ceruleo',
+  'high-desert',
+  'melon',
+  'plum',
+  'sunflower',
+  'verdant-haze',
+  // Numbered themes (experimental - not yet named)
+  '101', '102', '103', '104', '105', '106', '107', '108', '109', '110',
+  '111', '112', '113', '114', '115', '116', '117', '118', '119', '120',
+  '121', '122', '123', '124', '125', '126', '127', '128', '129', '130',
+  '131', '132', '133', '134', '135', '136', '137',
```

Comment:
Could be this replaced by `...NUMBERED_THEMES` ?

It can be outdated if it is not updated correctly.